### PR TITLE
fix(ComponentRelation): fix visualization on details page

### PR DESCRIPTION
**Pull request by sahil143**
## Fixes 

https://issues.redhat.com/browse/KFLUXBUGS-1106

## Description

fix component relation visualization on components details page

## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

image
image
image

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by karthikjeeyar** [comment]
The arrow position seems little off in chrome, it works fine in firefox though.

image

**Comment by sahil143** [comment]
> The arrow position seems little off in chrome, it works fine in firefox though.
> 
> image

@karthikjeeyar should be fixed now

File: src/components/ComponentRelation/details-page/ComponentNudgesDependencies.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import url(style8.css);
```
Comment (karthikjeeyar) [inline src/components/ComponentRelation/details-page/ComponentNudgesDependencies.scss:1]: I dont see the usage, is it needed?

**Pull request merged**